### ISSUE TITLE
Nutch 2.x support

### ISSUE THREAD

**Issue by tomchiverton** [BayanGroup/nutch-custom-search#41]

**Comment by kaidul** [comment]
hi, I've implemented Nutch 2.x version of this plugin and tested on my project. Can I send a pull request?

**Comment by haidawyl** [comment]
@kaidul Could you show me your code? I want  Nutch 2.x version.

**Comment by kaidul** [comment]
@haidawyl Which functionality do you need? I wrote some plugins recently but didn't furnish enough to send the PR here. Let me knowwhich functionality do you  need and I will send you the code personally.

**Comment by haidawyl** [comment]
@kaidul I can't find org.apache.nutch.parse.HtmlParseFilter in Nutch 2.x version.

**Comment by haidawyl** [comment]
@kaidul I hope all of them. When I change Nutch to 2.3.1, there are errors in ExtractorFetchSchedule.java, ExtractorIndexingFilter.java, ExtractorParseFilter.java, ExtractorParser.java, ExtractorScoringFilter.java, and OPICScoringFilter.java, I could not compile them.

**Comment by kaidul** [comment]
Yes, it isn't supposed to work that way for Nutch 2.x I meant what functionality do you require for your project as you're planning to use this plugin?

**Comment by bl4ck1c3** [comment]
Is this plug-in still available for 1.x?

Thank you

**Comment by haidawyl** [comment]
@kaidul Could you send the project 'zal.extractor.nutch' to me(haidawyl@163.com).

Thank you.

**Comment by haidawyl** [comment]
@kaidul I am trying to implement Nutch 2.x version, but it is hard to me.

**Comment by haidawyl** [comment]
@kaidul I could extract fields from the pages.
But how to store the values to hbase. It does not append fields to org.apache.nutch.storage.WebPage.
Could you help me? Thanks.

**Comment by hainguyenvan** [comment]
Hi haidawyl
You read code plugin *"parse-selector"* link git hub :
*https://github.com/hainguyenvan/apache-nutch-2.3/tree/master/src/plugin/parse-selector
<https://github.com/hainguyenvan/apache-nutch-2.3/tree/master/src/plugin/parse-selector>
*in me implement parse data by select xpath and save data to hase. Index
data to
elasticsearch you config you config *index.metadata* in *nutch-default.xml*
Thank !

**Comment by haidawyl** [comment]
@hainguyenvan Thanks! I will read the source.
